pkgs/desktops/gnome/core/gnome-initial-setup/default.nix

```diff
@@ -77,11 +78,12 @@ stdenv.mkDerivation rec {
     libnma
     libpwquality
     librest
+    librest_1_0
```

Comment:
Why are both needed?

Reply:
Have not looked into it yet. Most likely it is in requires of some other dependency, I just added it temporarily to make it build and forgot to add a comment.